### PR TITLE
Fix and enhance postman endpoints

### PR DESCRIPTION
**Pull request by ruilopesm**
This PR fixes and enhances almost all postman endpoints.

I would like to know your opinion on the below points:

1. I think it would be a nice addition to have, for testing purposes, authentication endpoints with different roles (`organizer`, `guardian`and `ninja`, for instance).
2. I also think it would be a nice addition, in order to better track massive changes like this one, to implement [newman](https://github.com/postmanlabs/newman) as a GitHub action.

### CODE REVIEW

**Comment by ruioliveira02** [comment]
I agree with both your points. The Github action would better be implemented in another PR, however I think the different authentication endpoints could be added right now.

File: .postman/collection.json [hunk 1]
```diff
@@ -3807,22 +4440,38 @@
...
-									"raw": "{{host}}/api/lectures/{{lecture_id}}",
+									"raw": "{{host}}/api/lectures/6fae868f-c214-440d-a03a-a20d1881f3be",
 									"host": [
 										"{{host}}"
 									],
 									"path": [
 										"api",
 										"lectures",
-										"{{lecture_id}}"
+										"6fae868f-c214-440d-a03a-a20d1881f3be"
```
Comment (nelsonmestevao) [inline .postman/collection.json:4855]: I think it should be a dynamic `lecture_id` here.

Comment (ruilopesm) [reply .postman/collection.json:4855]: I was testing and forgot to change it back 😅

**Review by nelsonmestevao — APPROVED** [review]
Looking forward for a GitHub Action with newman 🙏

**Pull request merged**